src/coreclr/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets

```diff
@@ -125,17 +125,23 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       <_KeyFileArgument Condition="'$(KeyOriginatorFile)' != ''">-KEY="$(KeyOriginatorFile)"</_KeyFileArgument>
 
+      <_IlasmCommand>$(_IlasmDir)ilasm</_IlasmCommand>
+      <_IlasmCommand Condition="'$(OS)'=='Windows_NT'">"$(_IlasmCommand)"</_IlasmCommand>
+
+      <_IlasmResourceFilePath>$(IlasmResourceFile)</_IlasmResourceFilePath>
+      <_IlasmResourceFilePath Condition="'$(OS)'=='Windows_NT'">"$(_IlasmResourceFilePath)"</_IlasmResourceFilePath>
```

Comment:
Double quotes for path with spaces is typically also required on Unix.  Maybe just add quotes directly to line 135 and 138 in the original revision.

Reply:
#13766 reported that adding quotes directly may break Unix build, thus I made this pull request targeting towards Windows first.